atlasdb-config/src/main/java/com/palantir/atlasdb/factory/ServiceDiscoveringAtlasSupplier.java

```diff
@@ -68,7 +71,7 @@ public KeyValueService getKeyValueService() {
     }
 
     public synchronized TimestampService getTimestampService() {
-        DebugLogger.logger.info("Fetching timestamp service from thread {}. This should only happen once.",
+        log.info("Fetching timestamp service from thread {}. This should only happen once.",
```

Comment:
This log is for tracking down problems arising from https://github.com/palantir/atlasdb/issues/1000 too! As per the JavaDoc, shouldn't enabling `Debuglogger` help catch these messages as well? I guess the idea behind using the `DebugLogger` here was to keep all the logs related to MTRSE together and being able to redirect that to a file if needed, changing the logger here will decrease value of the logging, i.e. we will have to look for multiple kinds on logs to track the issue down.